backend/src/libs/interfaces/repository.interface.ts

```diff
@@ -1,5 +1,6 @@
 interface IRepository<T = unknown> {
-  find(payload: unknown): Promise<T>;
+  find(id: number): Promise<T>;
```

Comment:
i think it should be id:unknown, email: unknown, check create method payload parameter